sigmf-spec.md

```diff
@@ -551,26 +551,26 @@ Example `top-level.sigmf-collection` file:
 
 ## Licensing
 
-Open licenses are recommended but you can specify any license. You can refer to 
+Open licenses are RECOMMENDED but you can specify any license. You can refer to 
 resources provided by the [Open Data Commons](https://opendatacommons.org/) when 
 deciding which open license fits your needs best. Cornell University has also 
 created [a guide](https://data.research.cornell.edu/content/intellectual-property#data-licensing) 
 to help you make these choices.
 
 ## SigMF Compliance by Applications
 
-In order to be `SigMF Compliant`, an application must meet the following
+In order to be `SigMF Compliant`, an application MUST meet the following
 requirements:
 
 1. Adheres to and supports the file rules, dataset formats, `objects`,
    `namespaces`, and `names` specified by this document.
-2. Must be able to ignore any `object` or `namespace` not specified by this
+2. MUST be able to ignore any `object` or `namespace` not specified by this
```

Comment:
I can imagine a SDR that might like to market itself as "SigMF Compliant" but may require special fields like a transmit frequency(?).

Imagine a really low-cost SDR that only supports a subset of the spec -> like a single channel recording or particular endianness.

Reply:
Are you advocating that this line should be changed to something like 

```
2. MUST be able to function normally without relying on any `object` or `namespace `not specified by this document or a required SigMF extension.
```